res/css/views/elements/_RichText.scss

```diff
@@ -14,8 +14,10 @@
 }
 
 a.mx_Pill {
-    word-break: break-all;
-    display: inline;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    max-width: 100%;
```

Comment:
I think we also need something like `vertical-align: text-bottom;`, when trying this out the alignment seems off in chrome:

![image](https://user-images.githubusercontent.com/274386/79562394-8e5d0080-809a-11ea-882b-d911e0862f75.png)

Reply:
Cool, will add. Will also experiment with `max-width: calc(100% - 1ch);` to allow for the `:` to be on the same line

Reply:
![image](https://user-images.githubusercontent.com/2403652/79564783-34166c80-80a7-11ea-9af6-8bfbf7a60b21.png)